engine/src/main/java/org/destinationsol/game/screens/ChooseMercenaryScreen.java

```diff
@@ -31,7 +31,7 @@
 //    private final SolUiControl equipControl;
     private final ItemContainer EMPTY_ITEM_CONTAINER = new ItemContainer();
 
-    ChooseMercenaryScreen(InventoryScreen inventoryScreen, GameOptions gameOptions) {
+    ChooseMercenaryScreen() {
```

Comment:
Any reason to have this empty constructor?

Reply:
It contains some commented lines from the previous implementation that could be useful when fully porting over the screen. They can be removed though.